ui/tests/integration/components/shamir-modal-flow-test.js

```diff
@@ -0,0 +1,103 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, find, pauseTest, click } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+
+module('Integration | Component | shamir-modal-flow', function(hooks) {
+  setupRenderingTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.set('isActive', true);
+    this.set('onClose', sinon.spy());
+  });
+
+  test('it renders with initial content by default', async function(assert) {
+    // Set any properties with this.set('myProperty', 'value');
+    // Handle any actions with this.set('myAction', function(val) { ... });
+
+    await render(hbs`
+      <div id="modal-wormhole"></div>
+      <ShamirModalFlow
+        @action="generate-dr-operation-token"
+        @buttonText="Generate token"
+        @fetchOnInit=true
+        @generateAction=true
+        @buttonText="Crazy CTA"
```

Comment:
this is a nit, but in the spirit of inclusive language would you mind swapping out 'crazy' for something else? aside from removing all references to blacklist/whitelist in our products and codebase, it'd be great to remove 'crazy' 'sanity check' 'insane' etc oo. :)